src/App/App.tsx

```diff
@@ -143,6 +146,14 @@ import { oAuthUserInfoGetter } from '../helpers/utils';
 import store from '../store';
 import './App.css';
 
+/** register reducer for configs */
+reducerRegistry.register(configsSliceName, configsReducer);
+const opensrpLanguageConfig = {
+  languageCode: LANGUAGE as LanguageCode,
+};
+/** Dispatch language to use for opensrp packages */
+store.dispatch(addConfigs(opensrpLanguageConfig));
+
```

Comment:
@kelvin-muchiri here unfortunately the workflow here, requires we dispatch the configs before importing the components, we should hoist this functionality to the very top of this file.